nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-stateless/README.md

```diff
@@ -38,19 +41,19 @@ Where the arguments dictate the runtime to use:
 
 ### Examples:
 ```
-1) docker run --rm -it nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
-    RunFromRegistry Once --file /Users/nifi/nifi-stateless-configs/flow-abc.json
+1) ${NIFI_HOME}/bin/nifi.sh stateless RunFromRegistry Once --file /Users/nifi/nifi-stateless-configs/flow-abc.json
 2) docker run --rm -it nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
+    RunFromRegistry Once --file /Users/nifi/nifi-stateless-configs/flow-abc.json
+3) docker run --rm -it nifi-stateless:1.10.0-SNAPSHOT-dockermaven \
```

Comment:
When I attempt to run like this I receive the following error:

> $ docker run --rm -it apache/nifi-stateless:1.10.0-SNAPSHOT-dockermaven RunFromRegistry Once --file ~/flow.json
> Exception in thread "main" java.lang.reflect.InvocationTargetException
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:498)
> 	at org.apache.nifi.bootstrap.RunStatelessNiFi.main(RunStatelessNiFi.java:69)
> Caused by: java.lang.reflect.InvocationTargetException
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:498)
> 	at org.apache.nifi.StatelessNiFi.main(StatelessNiFi.java:103)
> 	... 5 more
> Caused by: java.nio.file.NoSuchFileException: /Users/mgilman/flow.json

Do I need to do anything for the docker image to be able to access my flow.json? If so, we should probably document it here.

Reply:
fixed